app.js

```diff
@@ -112,15 +112,18 @@ app.use(function(req, res, next) {
 app.use(function(err, req, res, next) {
     var stacktrace = '';
     if (app.get('env') === 'development') {
-        // send stacktrace in development mode
         stacktrace = err.stack;
-        console.log("STACK:",err.stack);
+        console.error("ERROR:",err.message);
+        if(err.status !== 404) {
```

Comment:
I think the stack trace could only be shown for all 500-level errors. Perhaps here we could do:
```javascript
if (err.status >= 500 && err.status < 600)
```
All 400-level messages will be explicitly handled and should therefore be descriptive enough to not require a stack trace.